### PR TITLE
updated react to version 17

### PR DESCRIPTION
**Pull request by leonardodeutsch**
@kyelindholm @ks10825n 
- Updated react to version 17 to be compatible with fontawesome

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -57,14 +57,14 @@
...
     "@fortawesome/fontawesome-svg-core": "^1.2.36",
     "@fortawesome/free-brands-svg-icons": "^5.15.4",
     "@fortawesome/free-regular-svg-icons": "^5.15.4",
     "@fortawesome/free-solid-svg-icons": "^5.15.4",
-    "@fortawesome/react-fontawesome": "^0.1.16"
+    "@fortawesome/react-fontawesome": "^0.1.16",
+    "axios": "^0.24.0",
+    "moment": "^2.29.1",
+    "react": "^17.0.2",
+    "react-dom": "^15.6.0"
```
Comment (ks10825n) [inline package.json:68]: Can we actually stick with React v15? If we upgrade to React v17, we'd have to upgrade ReactDOM to match it. Then we'd have to reconfigure our testing libraries to work with v17. I initially downgraded to v15 to get the testing-library/react to work. We ended up going with the enzyme testing library instead. So we then have to use an enzyme-adapter-react-17 module that isn't officially supported.

Can you use connect to fontawesome via script in HTML?
something like this? `<script src="https://kit.fontawesome.com/ea1d6099b7.js" crossorigin="anonymous"></script>`

**Review by ks10825n — CHANGES_REQUESTED** [review]
Hey we'll have to change a few things if we want to upgrade to v17. Can we stick with v15? See comment for more details.

File: package.json (hunk 1 above)
Comment (ks10825n) [reply package.json:68]: Jk, I see the issue. You would want it in the react component. There should be a version compatible with React v15. I'll help you look

Comment (leonardodeutsch) [reply package.json:68]: ok, I did try to downgrade react-fontawesome but couldn't get it to work

Comment (ks10825n) [reply package.json:68]: Jk again. Just tried it out with a random icon you can just put 
`<script src="https://kit.fontawesome.com/ea1d6099b7.js" crossorigin="anonymous"></script>` into the head in the HTML file. Then when you find an icon you want to use, just put it in the render section of your component, and just change class to className and it should be good.

Comment (leonardodeutsch) [reply package.json]: @ks10825n 
ok got it working just like you said, thank you
dependencies are back to react 15 and I removed the fontawesome-react dependency that was causing the problem.

**Review by ks10825n — APPROVED** [review]
Thanks!

**Pull request merged**